README.md

```diff
@@ -15,37 +15,19 @@
     ·
     <a href="https://github.com/TheNerdful8/Lecture2Gether/pulls">Send a Pull Request</a>
   </p>
+  <p align="center">
```

Comment:
Maybe we should restructure the links at this point. At the moment the layout is like this:
```
    Demo - Bug report - Feature request - Pull request
            Screenshot A     -     Screenshot B
```
I think it is more consistent if we group the demo and the screenshots in one line and the other stuff in another line. Like this:
```
    Bug report  -  Feature request  -  Pull request
    Demo   -    Screenshot A     -     Screenshot B
```

I am not sure if the bug stuff or the demo stuff should be on the top.

Reply:
Nvm, I only saw the alt text as a link. But the screenshots are really embedded as an image. 

I have no concerns with embedding the screenshots in the `README.md`, but I don't like the placement at the top. It kinda breaks the layout there. 
I would favor a separate screenshots section under the `About` part. A caption could also be added to each screenshot.